### PR TITLE
feat(CAD): Enable CAD on signin for 50% of people.

### LINKED ISSUE #5721: CAD on signin is only enabled for mozilla/softvision addresses
Found by @ryanfeeley because of an article released about Quantum.

The dreaded green checkbox

### PR DESCRIPTION
**Pull request by shane-tomlinson**
We did all the groundwork to release CAD on signin
but forgot to pull the trigger!

fixes #5721

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@mozilla/fxa-devs - r? I pointed this at train-100 hoping we could use this to see a bump because of the Quantum release excitement.

**Comment by shane-tomlinson** [comment]
I forgot to say, we already have functional tests for [treatment/control groups](https://github.com/mozilla/fxa-content-server/blob/56e3a7eeca2276c9180211e1d4993642409ba6bf/tests/functional/sync_v3_sign_in.js#L96:L115), we just didn't ever actually use either of those groups except for with @mozilla & @softvision emails.

File: app/tests/spec/lib/experiments/grouping-rules/connect-another-device-on-signin.js [hunk 1]
```diff
@@ -8,36 +8,71 @@ define(function (require, exports, module) {
...
         assert.isFalse(experiment.choose());
         assert.isFalse(experiment.choose({}));
-        assert.isFalse(experiment.choose({ account: new Account() }));
-      });
 
-      it('returns true for test email addresses', () => {
-        account.set('email', 'testuser@testuser.com');
-        assert.isFalse(experiment.choose({ account }));
+        assert.isFalse(experiment.choose(Object.assign(validSubject, { env: undefined })));
+        assert.isFalse(experiment.choose(Object.assign(validSubject, { uniqueUserId: undefined })));
```
Comment (philbooth) [inline app/tests/spec/lib/experiments/grouping-rules/connect-another-device-on-signin.js]: Is this `Object.assign` doing what you want it to? I'd have thought you want to test each invalid property in isolation, but by modifying `validSubject` in place like this it seems like you're polluting subsequent assertions with the the target of the earlier ones. Or am I missing something?

Comment (shane-tomlinson) [reply app/tests/spec/lib/experiments/grouping-rules/connect-another-device-on-signin.js]: Oh dear lord, you are right!

**Comment by rfk** [comment]
> We did all the groundwork to release CAD on signin but forgot to pull the trigger!

Whoops :grin:

Do we not have a feature card etc for this one, to reminder us to roll it out?

**Comment by shane-tomlinson** [comment]
> Do we not have a feature card etc for this one, to reminder us to roll it out?

We do! https://projects.growthhackers.com/cards/add-connect-another-device-to-signin/ghpyETxuFuK63zGPZsC4kw We didn't follow up to discuss metrics like we said we would, or else we might've caught this earlier.

**Pull request merged**